Patches/CheckGameEndPatch.cs

```diff
@@ -612,11 +610,14 @@ public bool CheckGameEndByLivingPlayers(out GameOverReason reason)
                     reason = GameOverReason.ImpostorByKill;
                     CustomWinnerHolder.ResetAndSetWinner(CustomWinner.Impostor);
                 }
+
                 else if (impCount == 0) // Remaining Imps are 0, Crew wins (neutral is already dead)
                 {
                     reason = GameOverReason.HumansByVote;
                     CustomWinnerHolder.ResetAndSetWinner(CustomWinner.Crewmate);
                 }
+
+                else if (crewCount > impCount) return false;
                 return true;
             }
```

Comment:
When Crewmate turned out to be larger than Impostors and there were no specified reasons for end game, it always returned true, which could possibly cause many bugs with the end of the game